### PR TITLE
docs: typos and wording

### PR DESCRIPTION
**Pull request by vchimev**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/angular/angular/blob/main/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes/features)
- [ ] Docs have been added/updated (for bug fixes/features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no API changes)
- [ ] Build related changes
- [ ] CI-related changes
- [ ] Documentation content changes
- [x] angular.io application/infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: N/A

A few typos and wording in the quick start.

## What is the new behavior?

Update typos and wording in the quick start.

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

**Review by hexiaokang — APPROVED** [review]

File: aio/content/marketing/quick-start.md [hunk 1]
```diff
@@ -19,7 +19,7 @@ See more about [Angular features](/features) or try the following exercises for
 
 To give your component a reset button that sets the counter back to 0:
 
-1.  In the code editor, in the `template` value, after the line that starts with `<button (click)=`, add this line.
+1.  In the code editor, in the `template` value, add this line after the line that starts with `<button (click)=`.
```
Comment (bob-watson) [inline aio/content/marketing/quick-start.md:22]: Please revert this to the original.
I realize it sounds kind of clunky when read out loud, but the order is intentional.
The text that precedes the action is intended to guide the reader to the location of the action before presenting the action. This is to help prevent errors by people who implement the action as soon as they see it before they find the correct location.  Sadly, not every one reads every word of the documentation.
```suggestion
In the code editor, in the `template` value, after the line that starts with `<button (click)=`, add this line.
```

File: aio/content/marketing/quick-start.md [hunk 2]
```diff
@@ -55,7 +55,7 @@ You can also add styles to your new component.
 
     </code-example>
 
-1.  The buttons in your new Angular app should now have bold text and rounded corner.
+1.  The buttons in your new Angular app should now have the bold text and rounded corners.
```
Comment (bob-watson) [inline aio/content/marketing/quick-start.md:58]: Thanks for catching the typo.
The `the`, however, isn't necessary.
```suggestion
1.  The buttons in your new Angular app should now have bold text and rounded corners.
```

**Review by bob-watson — CHANGES_REQUESTED** [review]
Thanks for catching these typos!
There are a couple that I'd like reverted and for which I added some explanation.

Thanks, again for the contribution!

**Comment by vchimev** [comment]
Hey @bob-watson,

Thank you for the feedback! I reverted the requested changes.

**Review by jessicajaniuk — APPROVED** [review]

**Comment by jessicajaniuk** [comment]
caretaker: this is a low risk PR that's safe to merge without further approvals.

**Comment by jessicajaniuk** [comment]
This PR was merged into the repository by commit 35dabf986278b659c82eb67d0e13471462a5ae62.

**Pull request closed without merging**